### ISSUE TITLE
Remote access accessories

### ISSUE THREAD

**Issue by cdce8p** [ikalchev/HAP-python#94]
HomeKit supports devices accessing the accessories through hubs. This seems to work with `homebridge`, but not with `HAP-python`. Do you know if it's something we can fix or if it's on Apple's side?

Related:
https://github.com/home-assistant/home-assistant/issues/13981

**Comment by ikalchev** [comment]
Do you mean through an Apple TV/iPad? This used to work in HAP-python, but haven’t tested recently. Is this what you mean?

**Comment by cdce8p** [comment]
Yes. In the case that your iPhone isn't in the Wifi network, it should still be possible to contol accessories if you have an active hub.

I tested it today with an iPad Air 2 as hub, but that didn't work. The accessories stayed unresponsive. It was the same for `homebridge` which is why I belive that it is a issue Apple has to fix. I was just curious if aou know anything about it.

**Comment by ikalchev** [comment]
I will be able to test tomorrow. I can confirm that this worked a few weeks ago for me and one other user I know of.
Will get back to you when I test

**Comment by ikalchev** [comment]
https://9to5mac.com/2017/12/07/homekit-vulnerability/123/ which iOS versions have you tested?

**Comment by cdce8p** [comment]
11.3
